src/main/kotlin/lotto/domain/WinningStatistics.kt

```diff
@@ -0,0 +1,37 @@
+package lotto.domain
+
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+typealias MatchedCount = Int
+
+class WinningStatistics(private val winningLotto: WinningLotto) {
+    private lateinit var statistics: Map<Rank, MatchedCount>
+
+    fun calculateStatistics(purchasedLottos: PurchasedLottos): WinningStatistics {
+        statistics =
+            purchasedLottos
+                .calculateRanks(winningLotto)
+        return this // 메서드 체이닝 지원
+    }
+
+    fun count(rank: Rank): Int {
+        return statistics[rank] ?: 0
+    }
```

Comment:
```suggestion
    fun count(rank: Rank): Int = statistics[rank] ?: 0
```

바로 결과를 줄 수 있는 부분에 대해 식과 문 중에 무엇을 더 선호하시나요?

Reply:
식이 더 좋습니다! 
아직 자바의 습관이 ㅎㅎ 😓